challenges/classes.js

```diff
@@ -1,11 +1,34 @@
 // Copy and paste your prototype in here and refactor into class syntax.
 
+class CuboidMaker {
+  constructor (cuboid) {
+    this.width=cuboid.width;
+    this.height=cuboid.height;
+    this.length=cuboid.length;
+  }
+
+  volume () {
+    var vol = (this.height*this.length*this.width);
+    return vol;
+    console.log(vol);
+  };
+  surfaceArea () {
+    var sarea = 2*( (this.length*this.width) + (this.length*this.height) + (this.width * this.height) );
+    return sarea;
+    console.log(sarea);
+  };
+}
+
+var cuboid = new CuboidMaker ({length:4, width:5, height:5});
+
 // Test your volume and surfaceArea methods by uncommenting the logs below:
-// console.log(cuboid.volume()); // 100
-// console.log(cuboid.surfaceArea()); // 130
+console.log(cuboid.volume()); // 100
+console.log(cuboid.surfaceArea()); // 130
```

Comment:
When in doubt when making your own code, console.log every time.
Pay attention to how these are laid out and keep it in mind when you go to do your own projects. It's nice to have an idea of best practices.